client/src/store/registerModalAction.js

```diff
@@ -0,0 +1,16 @@
+export const SHOW_REGISTER_MODAL = 'SHOW_REGISTER_MODAL'
```

Comment:
We use actionTypes.js file for such variables.

Reply:
+1 agree